src/Avalonia.Controls/Automation/Peers/ControlAutomationPeer.cs

```diff
@@ -117,15 +117,22 @@ protected override IReadOnlyList<AutomationPeer> GetOrCreateChildrenCore()
             return result;
         }
         protected override string? GetHelpTextCore()
-        {          
+        {
             var result = AutomationProperties.GetHelpText(Owner);
 
             if (string.IsNullOrWhiteSpace(result))
             {
                 result = ToolTip.GetTip(Owner) as string;
             }
 
-            return result;          
+            if (string.IsNullOrWhiteSpace(result))
```

Comment:
Shouldn't this take higher priority than the tooltip fallback above? Validation errors should be one of the first things shown.

Reply:
Yes, it makes more sense, since the error should be "urgent and must-fix". I've changed it